packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -58,11 +60,20 @@ export interface DashboardLayoutSlots {
    * @default Account
    */
   toolbarAccount?: React.JSXElementConstructor<AccountProps>;
+  /**
```

Comment:
Instead of creating any new slots, could we just move the `<Account />` component to be included by default on the right of the `toolbarActions` slot, and get rid of the `toolbarAccount` slot?

Then anyone could customize these actions to have whatever they wanted in any order.
Some slight updates might also be needed to the documentation around this slot.

Reply:
I'm unsure to simply 'drop' the toolbarAccount slot, shouldn't we deprecate it first?

Reply:
Actually yeah, that might be better, we can just deprecate it!

Reply:
@apedroferreira I just did that and added the documentation. Thanks for the help!

<img width="1415" alt="image" src="https://github.com/user-attachments/assets/03933686-1fd7-4ef0-a95f-2a9cdf71e7e9" />